docs/content/build-and-deploy.md

```diff
@@ -16,10 +16,10 @@ To do this, either:
 * In a terminal window, from the root of your game project, run:  `spatial build build-config` in the [`spatial` CLI](https://docs.improbable.io/reference/latest/shared/glossary#the-spatial-command-line-tool-cli).
```

Comment:
(1) I think Deployment needs its own page.
(2) I think the text at the top of the page should be something like:
**Deployment**
When you want to try out your game, you need to run a deployment of the game. This means launching SpatialOS itself. SpatialOS sets up the game world and starts up the workers needed to run the game world. Once the deployment is running, you can connect clients to it in order to play the game. You can run a deployment on your development machine (a "local deployment" or in the cloud (a "cloud deployment"). <Editor note - remove the links as not needed.>
Before you deploy your game, you need to set up its workers; see documentation on [building workers].
This page describes starting your game locally<link to this section of the page> or in the cloud <link to this section of the page>. You can also follow a walk-through of launching the `Playground` project <link to lower in the page> - a basic project which comes with the GDK.